packages/main/src/SegmentedButton.ts

```diff
@@ -289,10 +324,24 @@ class SegmentedButton extends UI5Element {
 	 * @readonly
 	 * @type {sap.ui.webc.main.ISegmentedButtonItem}
 	 * @name sap.ui.webc.main.SegmentedButton.prototype.selectedItem
+	 * @deprecated As of 1.14.0. This method is not guaranteed in future releases.
```

Comment:
is not guaranteed in future releases. -> will be removed in the next major release.

Because, we can't just remove it and release 1.15.0. If we remove it we need to go for 2.0

Reply:
Done